src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -50,3 +50,22 @@
 .right {
   grid-column: 4/5;
 }
+
+.filterLabel {
+  grid-column: 1/2;
+  text-align: right;
+  padding-right: 2px;
+}
+
+.expandBtn {
+  margin-left: 10px;
+
+  img {
+    height: 12px;
+    width: 12px;
+  }
+}
+
+.hidden {
+  display: none;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/84207693-df411200-aaa9-11ea-93af-9fd67d90ca61.png)

Notice this red sign? I believe it says that there is no empty line in the end of the file. Not that it's particularly important, but I believe our editors are set up in a way that they automatically add an empty line in the end of the file when you save it.

Reply:
It wasn't on in my editor, i have enabled it.